### PR TITLE
Extend CLI config to support on-premise URLs

### PR DESCRIPTION
**Pull request by baffo**
Use Netlify ENV variable SENTRY_URL or plugin input sentryUrl to set a custom on-premise url to report new release. Fallback to https://sentry.io/ if not specified.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ module.exports = {
     const IS_PREVIEW = process.env.CONTEXT == 'deploy-preview'
 
     /* Set the user input settings */
+    const sentryUrl = process.env.SENTRY_URL || inputs.sentryUrl || "https://sentry.io/";
```
Comment (NisanthanNanthakumar) [inline index.js:30]: @baffo by default in the sentry-cli we set the `SENTRY_URL` to "https://sentry.io/" if it is not set. https://github.com/getsentry/sentry-cli/blob/38f7f254c03f4e3681463881a2eb54266d08ea5e/src/config.rs#L545

I think it's best to have one source of truth for the defaults and use sentry-cli.

File: index.js [hunk 2]
```diff
@@ -128,11 +129,12 @@ async function createSentryRelease({ pluginApi, release, sentryEnvironment, sour
   await cli.releases.execute(['releases', 'deploys', release, 'new', '-e', sentryEnvironment])
 }
 
-async function createSentryConfig({ sentryOrg, sentryProject, sentryAuthToken }) {
+async function createSentryConfig({ sentryOrg, sentryProject, sentryAuthToken, sentryUrl }) {
   const sentryConfigFile = `
   [auth]
   token=${sentryAuthToken}
   [defaults]
+  url=${sentryUrl}
```
Comment (NisanthanNanthakumar) [inline index.js:137]: @baffo since this a template string, we can use a conditional to determine whether to include this line if not null

**Review by NisanthanNanthakumar — CHANGES_REQUESTED** [review]
Thanks for making the PR! Small changes required for approval.

File: index.js (hunk 1 above)
Comment (baffo) [reply index.js]: I agree with the suggestion @NisanthanNanthakumar, I removed the literal declaration of the url as it's unnecessary

File: index.js (hunk 2 above)
Comment (baffo) [reply index.js]: Thank you for the idea, I commited a conditional instead.

**Review by NisanthanNanthakumar — APPROVED** [review]
Looks good!

**Pull request merged**